test/unit/transporters/nats.spec.js

```diff
@@ -389,13 +389,13 @@ describe("Tests Nats V2.x", () => {
 		it("check constructor with string param", () => {
 			let transporter = new NatsTransporter("nats://localhost");
 			transporter.isLibLegacy = jest.fn(() => false);
-			expect(transporter.opts).toEqual({ preserveBuffers: true, maxReconnectAttempts: -1, servers: ["nats://localhost"] });
+			expect(transporter.opts).toEqual({ preserveBuffers: true, maxReconnectAttempts: -1, url: "nats://localhost" });
 		});
 
 		it("check constructor with string param of multiple servers", () => {
 			let transporter = new NatsTransporter("nats://server1:4222,nats://server2:4222,nats://server3:4222");
 			transporter.isLibLegacy = jest.fn(() => false);
-			expect(transporter.opts).toEqual({ preserveBuffers: true, maxReconnectAttempts: -1, servers: ["server1:4222", "server2:4222", "server3:4222"] });
+			expect(transporter.opts).toEqual({ preserveBuffers: true, maxReconnectAttempts: -1, url: "nats://server1:4222,nats://server2:4222,nats://server3:4222" });
```

Comment:
But the original issue was that it should be in `servers` and not `url`, wasn't it?

Reply:
this cannot be done at the constructor level, since the nats version is checked later and therefore I had to transfer the formation of servers to the connection only on nats 2.x
https://github.com/moleculerjs/moleculer/blob/issue-953/src/transporters/nats.js#L147

Reply:
ok, you are right.